### ISSUE TITLE
no auto detecting weapon.

### ISSUE THREAD

**Issue by sadow200** [CharlesDankoff/ultra_secret_scripts#3]
It seems that its not auto detecting when i switch weapon when no recoil is on.

Im runing my game is window borderless mode.
Res 1920x1080

**Comment by Bac3Phi** [comment]
did u try to press **NUM_4** and **NUM_6**?

**Comment by sadow200** [comment]
yes i can change them manually but i was wondering why the auto detect doesn't work

**Comment by Bac3Phi** [comment]
when you start app `python ./very_secret_script.py apex`
did u get any errors? i think your computer missing some lib like **win32api**

**Comment by sadow200** [comment]
no errors everything else works fine just when i tunr it on and change weapons the script doesnt auto change to the correct weapon im using

**Comment by victorhramos** [comment]
no auto detecting here also

**Comment by Ascarre15** [comment]
Image file corrupted or not view our detected properly by the code update the pictures
